src/tasks/task.ts

```diff
@@ -70,7 +70,6 @@ export class Task implements TaskBase {
          */
         public readonly timestamp?: Date,
         /**
-         * @hidden

```

Comment:
Is there a reason we removed this hidden value?

Reply:
Seems like we're ready to merge @aaronpowell! Do you mind chiming in on this comment so I can finalize this PR? Thanks!

Reply:
Added back in